build.sh

```diff
@@ -364,64 +385,69 @@ while [[ $# -gt 0 ]]; do
     shift
 done
 
-if [[ $SHOULD_EMBED_ICU == 1 ]]; then
-    if [ ! -d "${CHAKRACORE_DIR}/deps/icu/source/output" ]; then
-        ICU_URL="http://source.icu-project.org/repos/icu/icu/tags/release-57-1"
-        echo -e "\n----------------------------------------------------------------"
-        echo -e "\nThis script will download ICU-LIB from\n${ICU_URL}\n"
-        echo "It is licensed to you by its publisher, not Microsoft."
-        echo "Microsoft is not responsible for the software."
```

Comment:
AFAIK; we should not update the language here.

Reply:
I just saw that you moved into py file. 👍

Reply:
I think I copied it directly aside from renaming ICU-LIB to ICU, since I have never heard it called ICU-LIB before.